includes/class-buoy-alert.php

```diff
@@ -1038,24 +1062,36 @@ public static function enqueueFrontEndScripts () {
         wp_register_script(
             self::$prefix.'-script',
             plugins_url(self::$prefix.'.js', dirname(__FILE__)),
-            array('jquery'),
+            array(
+                'jquery',
+                'underscore',
+                'backbone',
+            ),
             $plugin_data['Version']
         );
-        wp_localize_script(self::$prefix.'-script', self::$prefix.'_vars', self::localizeScript());
+        // Localize it.
+        wp_localize_script(self::$prefix.'-script', self::$prefix.'_vars', self::localizeUiText());
+        wp_localize_script(self::$prefix.'-script', self::$prefix.'_dom_hooks', self::$dom_hooks);
```

Comment:
Would you mind explaining this for my wordpress education? I was able to understand from the WP codex why the ui text would go through `wp_localize_text`, but I don't totally understand why the dom hooks thing is in here.